.env.test

```diff
@@ -0,0 +1,47 @@
+SECRET_KEY=insecure-secret-key
+LOGIN_CODE_SECRET=insecure-login-secret-key
+
+REL_BASE_DIR="../../"
+APP_ENV=test
+PUBLIC_BASE_URL=http://localhost:8087
+
+ADMIN_WEBSITE=https://www.thunderbird.net
+ADMIN_CONTACT=dummy@example.org
+SUPPORT_CONTACT=dummy@example.org
+
+# From Appointment
+FXA_OPEN_ID_CONFIG=https://accounts.stage.mozaws.net/.well-known/openid-configuration
+FXA_OAUTH_SERVER_URL=https://oauth.stage.mozaws.net/v1
+FXA_PROFILE_SERVER_URL=https://profile.stage.mozaws.net/v1
+FXA_CLIENT_ID=
+FXA_SECRET=
+FXA_CALLBACK=http://localhost:8087/api/v1/auth/fxa/callback
+FXA_ALLOW_LIST=
+FXA_ENCRYPT_SECRET=insecure-fxa-secret-key
+
+# Database
+DATABASE_NAME=accounts_test
+DATABASE_USER=accounts
+DATABASE_PASSWORD=accounts
+DATABASE_HOST=sqlite
+
+# Cache
+REDIS_URL=redis://redis:6379
+
+# Auth scheme
+AUTH_SCHEME=fxa
+
+IMAP_HOST=localhost
+IMAP_PORT=993
+IMAP_TLS=True
+
+SMTP_HOST=localhost
+SMTP_PORT=465
+SMTP_TLS=True
+
+JMAP_HOST=localhost
+JMAP_PORT=7768
+JMAP_TLS=True
+
+# MailChimp form URL for Wait List
+WAIT_LIST_FORM_ACTION=
```

Comment:
The README should probably be updated, currently it says to

`cp .env.example .env.test`

and

`In `.env.test`, set `APP_ENV=test``

But we don't need those now since you've added the file here 👍